frontend/src/metabase/actions/containers/ActionCreator/tests/ActionCreator-Common.unit.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import nock from "nock";
+import userEvent from "@testing-library/user-event";
+
+import { screen } from "__support__/ui";
+
+import {
+  createMockImplicitQueryAction,
+  createMockQueryAction,
+} from "metabase-types/api/mocks";
+
+import { setup as baseSetup, SetupOpts } from "./common";
+
+async function setup({
+  action = createMockImplicitQueryAction(),
+  ...opts
+}: SetupOpts = {}) {
+  await baseSetup({ action, ...opts });
+  return { action };
+}
+
+describe("ActionCreator > Common", () => {
+  afterEach(() => {
+    nock.cleanAll();
+  });
+
+  describe.each([
+    ["query", createMockQueryAction],
+    ["implicit", createMockImplicitQueryAction],
+  ])(`%s actions`, (_, getAction) => {
```

Comment:
`getAction` isn't called with any dynamic parameters in the test. Would it be more straightforward to simply pass an object instead, e.g. `["query", createMockQueryAction()]`

Reply:
I see. FWIW there's another similar pattern in `ActionCreator-Sharing.unit.spec.tsx` which actually passes an argument to `getAction`.

I'm fine either way whether or not this will be changed.